demos/react-dapp/src/App.tsx

```diff
@@ -800,6 +883,22 @@ const App: React.FC = () => {
             </button>
           </div>
         </section>
+        <section>
+          <h2>Signature Verification Test</h2>
+          <div>
+            <button
+              onClick={() => {
+                modalWrapper(async () => {
+                  if (!dAppConnector) throw new Error('DAppConnector is required')
+                  if (!selectedSigner) throw new Error('Selected signer is required')
+                  return handleTestSignatureVerification()
+                })
```

Comment:
`handleTestSignatureVerification()` is wrapped in a modalWrapper already. Do we need it wrapped again here inline?